docs/README.md

```diff
@@ -2,7 +2,40 @@
 
 _Our documentation is still a work in progress. There are a number of topics listed under [Features](#features) below that don't have outgoing links yet, but we've included them to give a better sense of the current feature set. Please open an issue if you'd like to know more about one of the topics we haven't gotten around to documenting yet._
 
-## Getting started 
+## What is .NET Interactive?
+
+.NET Interactive is an engine that can run multiple languages and share variables between them. Languages currently supported include: 
+
+- C# 
+- F#
+- PowerShell
+- JavaScript
+- SQL 
+- KQL (Kusto Query Language)
+- HTML*
+- Mermaid*
+
+*Variable sharing not available
+
+## What can .NET Interactive be used for? 
+
+As a powerful and versatile engine, .NET Interactive can be used to create and power a number of tools and experiences such as: 
+
+- Polyglot (multi-language) Notebooks
```

Comment:
If we're using `Polyglot Notebooks` here to refer to the product name, then putting the clarifier in the middle is probably not appropriate.

Also, should this link to the VS Code Marketplace page for the extension?